components/Participants.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { usePresence, assertConfiguration } from "@ably-labs/react-hooks";
+import styles from "../styles/Home.module.css";
+
+export default function Participants(props) {
+  const ably = assertConfiguration();
```

Comment:
assertConfiguration() returns the current configured instance of the Ably SDK that the react hooks package uses, called here to prevent it having to be passed as a prop to the components. 
Much easier to just have it where we need it.